includes/core/server/request/class-refund-charge.md

```diff
@@ -0,0 +1,33 @@
+# `Refund_Charge` request class
+
+[ℹ️ This document is a part of __WooCommerce Payments Server Requests__](../requests.md)
+
+## Description
+
+The `WCPay\Core\Server\Request\Refund_Charge` class is used to construct the request for refunding a charge.
+
+## Parameters
+
+When creating `Refund_Charge` requests, the item ID must be provided to the `::create()` method. The identifier should be in the `ch_XXX` format.
```

Comment:
I think we would remove this line like what did for `List_Charge_Refunds`.

Reply:
Addressed in https://github.com/Automattic/woocommerce-payments/pull/5967/commits/a688efe8c9c9b7853d0b10c2d2ba089cb3fa48f7